eth/transactionManager.go

```diff
@@ -50,7 +50,8 @@ type TransactionManager struct {
 type transactionQueue []*types.Transaction
 
 type transactionReceipt struct {
-	*types.Receipt
+	txOriginHash ethcommon.Hash
```

Comment:
```suggestion
	origTxHash ethcommon.Hash
```

to avoid any confusion with the other common meaning of "tx origin" which refers to the origin of a tx within a smart contract.

Reply:
Right but that's an EVM specific and pertains to an address (20 bytes) not a hash (32 bytes) , but sure I don't feel strongly about naming here.